promql/functions.go

```diff
@@ -938,6 +936,21 @@ var FunctionCalls = map[string]FunctionCall{
 	"year":               funcYear,
 }
 
+// AtModifierUnsafeFunctions are the functions whose result
+// can vary if evaluation time is changed when the arguments are
+// step invariant. It also includes functions that use the timestamp
+// of the passed argument to calculate a result since that can
```

Comment:
The timestamps of passed instant vector arguments.

Range vector timestamps aren't an issue per-se (e.g. rate uses the timestamps of its arguments).


It's important to be precise with this sort of thing, it's hard enough to get these semantic correct without comments containing misleading information.